en/docs/integrations/integration-concepts.md

```diff
@@ -0,0 +1,98 @@
+# Concepts
+
+Before you start building integrations with Choreo, there are basic concepts that you need to understand. This section walks you through all the integration concepts you need to know. 
+ 
+## Integration
+
+Integration in Choreo is the process of connecting APIs, events, and streams to function as a single unit that can share information.
+
+
```

Comment:
Remove Extra new lines. Check all the places

Reply:
Done.